src/helpers/helpers.options.ts

```diff
@@ -120,7 +126,7 @@ export function toFont(options, fallback) {
   let style = valueOrDefault(options.style, fallback.style);
   if (style && !('' + style).match(FONT_STYLE)) {
     console.warn('Invalid font style specified: "' + style + '"');
-    style = '';
+    style = 'inherit';
```

Comment:
not sure about this one

Reply:
try to restore it, or try `undefined`

```suggestion
    style = undefined;
```

maybe this will fix tests

Reply:
empty string wont work, tried `undefined` but ubuntu test is still broken